easydiffraction/Job.py

```diff
@@ -632,16 +632,16 @@ def show_crystal_structure(self, id=None):
         phase = self.phases[id]
         cif = phase.cif
 
-        structure_view = py3Dmol.view(linked=False)
+        structure_view = py3Dmol.view(width=540, height=480, linked=False)
```

Comment:
It seems that in py3dmol the image size is hardcoded anyway. The default is 640x480, which isn't much bigger. The problem is that it's a bit out of place in the automatically generated html document. So I decided to shrink it down a bit. It would be nice to have something that dynamically adjusts to the page width like in Plotly, but py3dmol doesn't have that.